src/migrations/20220127220907-create-table-users.js

```diff
@@ -28,6 +28,10 @@ module.exports = {
           type: Sequelize.STRING,
           allowNull: false,
         },
+        roles: {
+          type: Sequelize.ENUM,
+          values: ["user", "admin", "superadmin", "developer"],
+        },
```

Comment:
We finally resolved to go with another direction in this matter. Users roles should be of type STRING, with a default value of "user". This was fixed in branch `OT124-97` so i suggest we discard this changes.